zerver/lib/send_email.py

```diff
@@ -25,6 +26,12 @@ class FromAddress:
     SUPPORT = parseaddr(settings.ZULIP_ADMINISTRATOR)[1]
     NOREPLY = parseaddr(settings.NOREPLY_EMAIL_ADDRESS)[1]
 
+    @staticmethod
+    def tokenized_no_reply_address() -> str:
```

Comment:
Can we add a couple tests that directly use this method?  I think the tests would be helpful not only to prevent regressions but also to illustrate what this does.  It's also kind of strange for a `tokenized_foo` message to return a `str`; I'd normally expect a "tokenized" thing to bee a list of strings.  So, it's possible the name here is a bit misleading, although it's also fine to just have a comment explain what it's doiing.